### ISSUE TITLE
How to align text left

### ISSUE THREAD

**Issue by butgit** [edrlab/thorium-reader#1230]
I see 'alignment' in the settings menu, but it has only two options: automatic and justify, and neither one makes the text left-aligned.
I find it hard to read justified text because it makes uneven the spacing between words. 
I'd appreciate if someone could tell me where I can find the relevant settings file. 
Thank you.

**Comment by divenex** [comment]
This looks like a bug indeed. In v1.5 on Windows 10, the icon for "automatic" text alignment shows a left-aligned text. But switching between automatic and justified does nothing. Users should be able to choose between justification and left/right alignment.

**Comment by danielweck** [comment]
Totally agree. We also discussed "auto" (default, which users should be able to reset to) for a number of other presentation properties, such as scroll vs. pagination.

**Comment by danielweck** [comment]
Fixed via https://github.com/readium/r2-navigator-js/commit/aaaddf51d2c58b4e8b61da6eb5a9dc9a361ca057
and https://github.com/edrlab/thorium-reader/commit/5f6fe81212cd5b6093255be668514e18542e6823
